tests/v2/integration/projects/resource_quota_test.go

```diff
@@ -1,18 +1,29 @@
 package integration
 
 import (
+	"context"
+	"fmt"
 	"testing"
-	"time"
 
 	"github.com/rancher/rancher/tests/framework/clients/rancher"
 	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/kubeapi/resourcequotas"
 	"github.com/rancher/rancher/tests/framework/extensions/namespaces"
-	"github.com/rancher/rancher/tests/framework/extensions/resourcequotas"
+	steveResourceQuotas "github.com/rancher/rancher/tests/framework/extensions/resourcequotas"
 	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/rancher/rancher/tests/framework/pkg/wait"
+	"github.com/rancher/rancher/tests/integration/pkg/defaults"
 	"github.com/stretchr/testify/suite"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+const (
+	resourcdQuotaNamespaceName  = "ns1"
```

Comment:
Typos with `resourcdQuotaNamespaceName` and r`esourcdQuotaNamespaceName2` - should be `resourceQuotaNamespaceName` and 2.